vllm/engine/async_llm_engine.py

```diff
@@ -293,6 +293,44 @@ async def process_model_inputs_async(
 
         return self.input_processor(llm_inputs)
 
+    async def process_model_params_async(
```

Comment:
Currently, async method is not tested. Checked only sync version and made async one based on it.